phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -178,6 +178,7 @@
 import org.apache.phoenix.trace.util.Tracing;
 import org.apache.phoenix.util.ByteUtil;
 import org.apache.phoenix.util.CursorUtil;
+import org.apache.phoenix.util.IndexUtil;
```

Comment:
nit: seems like you can revert this file

Reply:
Done